cozy-registry.example.yml

```diff
@@ -38,10 +38,17 @@ swift:
   auth_url: http://localhost:1234
   username: admin
   api_key: s3cret!
-  endpointy_type: public
+  endpoint_type: public
   tenant: demo
   domain: default
 
+# Deprecation configuration allows to manage paramaters of how many app versions
+# of each channel must be kept during cleaning tasks
+deprecation:
```

Comment:
[nit] I don't deprecation is the right term for that. Maybe cleaning or conservation